sql/src/main/java/io/crate/metadata/PartitionInfos.java

```diff
@@ -63,14 +63,17 @@ public PartitionInfo apply(@Nullable ObjectObjectCursor<String, IndexMetaData> i
                 PartitionName partitionName = PartitionName.fromIndexOrTemplate(input.key);
                 try {
                     MappingMetaData mappingMetaData = input.value.mapping(Constants.DEFAULT_MAPPING_TYPE);
+                    Map<String, Object> mappingMetaDataSourceMap = mappingMetaData.sourceAsMap();
```

Comment:
minor: its a map build from the source, so I'd call it `mappingMetaDataMap`, or just `mappingMap`